visualization_tool/USAGE.md

```diff
@@ -0,0 +1,66 @@
+# GCP Scanner Visualizer Usage Guide
+
+The GCP Scanner Visualizer is a web-based tool that can help you visualize the results of the GCP Scanner.
+
+## Running the tool
+
+To run the tool, you can use the following command `gcp-scanner-visualizer`. The tool will start a web server on port 8080. You can change the port by using the `-p` option.
+
+Then you can open your browser and navigate to `http://localhost:8080` to use the tool.
+
+![GCP Scanner Visualizer](../misc/visualization_tool_images/start_page.webp)
+
+## Uploading the results
+
+To upload the results, head to the `Upload` Section and click on the `Choose File` button. Then select the JSON file that you want to upload and click on the `Plus` icon.
+
+![GCP Scanner Visualizer](../misc/visualization_tool_images/upload.webp)
+
+After uploading the result file, the tool will scan the file and append the results to UI.
+
+You can also remove any file by clicking on the `minus` icon next to the file name.
+
+## Viewing the results
+
+There are two main pages in the tool:
+
+### Resources Page
+
+This main page and it shows the list of resources that were found in the result file. You can click on the `Details` button to see more details about the resource.
```

Comment:
`This main page` replace with `This is the main page`

Reply:
Done